easybuild/easyconfigs/c/cclib/cclib-1.6.3-foss-2020a-Python-3.8.2.eb

```
@@ -0,0 +1,39 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'PythonBundle'
+
+name = 'cclib'
+version = '1.6.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://cclib.github.io/'
+
+description = "parsers and algorithms for computational chemistry"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('periodictable', '1.5.2', {
+        'checksums': ['1fea8abf0c20453550630ca682a75f0148f65b6d21fdcce7cf0c0e98631fa0d3'],
+    }),
+    (name, version, {
+        'checksums': ['0428aa8c29b7460c767a8b3b14c089ee83bbd6269a0af2a2164eafa962f1c2d4'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ccframe', 'bin/ccget'],
+    'dirs': [],
```

Since you are defining some files in `sanity_check_paths`, the `site-packages` directory which would have been added by the `PythonBundle` easyblock is lost. Please, add it manually
```suggestion
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```